### PR TITLE
refactor(icons): add more indigo-specific icons

### PR DESCRIPTION
**Pull request by simeonoff**
Closes [#14856](https://github.com/IgniteUI/igniteui-angular/issues/14856)

### Additional information (check all that apply):
 - [ ] Bug fix
 - [x] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Comment by andiesm813** [comment]
Hey! I was reviewing this..... shouldnt these icons be the same? or maybe we need the indigo version for that date range calendar icon, which i didnt do. 
image

**Comment by yradoeva** [comment]
Here are the things I noticed:

1. **Grid GroupBy** - the block icon in the ghost header when dragging should be “indigo-block”
i material-icons igx-grid__drag-gh

2. **Grid Quick filter** - _that is something I somehow missed during previous reviews_
The chevron in the AND/OR operand button should be “indigo-chevron-down”
Screenshot 2024-10-04 at 9 52 28

<br/>
<br/>

And for the samples below I am not sure if they are just templates with custom icons, but wanted to mention them:
3. **Banner page - Navbar** - “menu” icon should be “indigo-menu”
Screenshot 2024-10-03 at 16 56 55

4. **Card**  - “Add” icon should be “indigo-add”
Rupert Stadler

5. **Forms - Date picker** - calendar icon should be “indigo-calendar-today” (the calendar dropdown here is also a custom one)
October 2024 -

6. **Grid filter template**
Search should be “indigo-search”
Clear should be “indigo-clear”
Screenshot 2024-10-04 at 10 10 43

7. **Hierarchical Grid**
Add icons should be “indigo-add”
Remove icons should be “indigo-horizontal-rule”
Pin should be “indigo-pin” (respectively “indigo-unpin”)
Screenshot 2024-10-04 at 9 46 00

8. **List** - Info icon should be “indigo-info”
Job positions

9. **Navbar** - “menu” icons should be “indigo-menu”, “search” icons should be “indigo-search”, “arrow_back” icon should be “indigo-arrow-back”
With custom nav icon

10. **Tooltip - Grid with tooltips** - Info icons should be “indigo-info”
Grid with tooltips

File: src/styles/_variables.scss [hunk 1]
```diff
@@ -1,10 +1,10 @@
 @use 'sass:map';
 @use '../../projects/igniteui-angular/src/lib/core/styles/themes' as *;
 
-$palette: $light-material-palette;
-$schema: $light-material-schema;
-$typeface: $material-typeface;
-$type-scale: $material-type-scale;
+$palette: $light-indigo-palette;
```
Comment (SisIvanova) [inline src/styles/_variables.scss:4]: I guess you pushed this change unintentionally.

**Review by SisIvanova — DISMISSED** [review]

File: src/styles/_variables.scss (hunk 1 above)
Comment (simeonoff) [reply src/styles/_variables.scss:4]: Correct.

**Comment by simeonoff** [comment]
> Hey! I was reviewing this..... Here are my comments.
> 
> ### **DATE RANGE CALENDAR ICON**
> Shouldnt these icons be more similar to the date picker custom indigo icon? Maybe we need the indigo version for that date range calendar icon, which i didnt do.
> 
> image
> 
> ### **GRID HEADER EXPAND / COLLAPSE ICONS**
> I think these need to be replaced with "indigo-add" and "indigo-horizontal-rule".
> 
> image
> 
> ### **ROW PINNING ICONS**
> I'm not sure i understand this sample, but shouldnt all the pin icons match the indigo pin / unpin? Or because in this scenario they are actually pinned to the left, they dont change?
> 
> image
> 
> ### **PAGINATOR**
> Are the icons on the top paginator supposed to be indigo custom icons? because they look different than the bottom paginator.
> 
> image
> 
> ### **TIME PICKER and DATE PICKER WITH CUSTOM SPIN BUTTONS**
> Since we already have these custom indigo icons too, shouldnt these be also using indigo-chevron-up / down? i know this is a custom spinner, but so they look similar to the other inputs which use indigo-chevron up/down.
> 
> image
> 
> ### **FLOATING ACTION BUTTONS**
> I think these should be replaced with "indigo-add" icon.
> 
> image
> 
> ### **SEARCH BOX**
> This should be using the indigo-search icon. Looks similar, but it's a bit different :)
> 
> image
> 
> ### **INPUT GROUP**
> Again, these are custom inputs i suppose, but i'm also wondering if the icons should be replaced with the custom indigo ones.
> 
> image
> 
> I think the rest look good and i did not notice any other icons left to replace with custom indigo ones. Good work! ;)

As discussed, most of the icons mentioned are part of custom templates in the demo samples. I've updated most of the ones that make more sense to be updated.

**Comment by andiesm813** [comment]
> Here are the things I noticed:
> 
> 1. **Grid GroupBy** - the block icon in the ghost header when dragging should be “indigo-block”
> 
> i material-icons igx-grid__drag-gh
> 2. **Grid Quick filter** - _that is something I somehow missed during previous reviews_
>    The chevron in the AND/OR operand button should be “indigo-chevron-down”
> 
> Screenshot 2024-10-04 at 9 52 28
> 
> And for the samples below I am not sure if they are just templates with custom icons, but wanted to mention them: 3. **Banner page - Navbar** - “menu” icon should be “indigo-menu” Screenshot 2024-10-03 at 16 56 55
> 
> 4. **Card**  - “Add” icon should be “indigo-add”
> 
> Rupert Stadler
> 5. **Forms - Date picker** - calendar icon should be “indigo-calendar-today” (the calendar dropdown here is also a custom one)
> 
> October 2024 -
> 6. **Grid filter template**
>    Search should be “indigo-search”
>    Clear should be “indigo-clear”
> 
> Screenshot 2024-10-04 at 10 10 43
> 7. **Hierarchical Grid**
>    Add icons should be “indigo-add”
>    Remove icons should be “indigo-horizontal-rule”
>    Pin should be “indigo-pin” (respectively “indigo-unpin”)
> 
> Screenshot 2024-10-04 at 9 46 00
> 8. **List** - Info icon should be “indigo-info”
> 
> Job positions
> 9. **Navbar** - “menu” icons should be “indigo-menu”, “search” icons should be “indigo-search”, “arrow_back” icon should be “indigo-arrow-back”
> 
> With custom nav icon
> 10. **Tooltip - Grid with tooltips** - Info icons should be “indigo-info”
> 
> Grid with tooltips

@simeonoff i saw some of the comments @yradoeva made were addressed.... and i assume the rest weren't because of the same response to my comments, correct? You also updated the ones that made sense, and kept the others unchanged regarding Dani's comments.

**Comment by andiesm813** [comment]
@yradoeva if Simo already change the icons that made more sense, i think everything else looks good to me.

**Pull request merged**